spec/factories/work_versions.rb

```diff
@@ -7,6 +7,11 @@
     title { generate(:work_title) }
     sequence(:version_number) { |n| work.present? ? work.versions.length + 1 : n }
 
+    after(:build) do |work_version|
+      # Postgres does this for us, but for testing, we can do it here to save having to call create/reload.
+      work_version.uuid = SecureRandom.uuid
+    end
+
```

Comment:
Rather than have this in a callback, could we just put this into a normal factorybot attribute?

```ruby
uuid { SecureRandom.uuid }
```

Reply:
@rschenk Sure. I had put in the `after(:update)` block to distinguish between `create` but I don't think that matters since calling "create" is going to execute the same things that `build` does up until the record is actually saved. I'm not 100% thrilled with this whole idea since we're bypassing Postgres, and making our test operations different than our dev and prod. operations, but I couldn't see another way, and I really didn't want to have to save/reload these test records all the time.